src/main/java/iitema/gypsypokemon/elements/ProjectileInterface.java

```diff
@@ -4,18 +4,22 @@
  * A projectile which can be shot
  */
 public interface ProjectileInterface {
+    /**
+     * Color of projectile
+     */
+    enum CartridgeColor {YELLOW, BLUE}
```

Comment:
We should move this out of the class, otherwise getColor() return value will not be available for anyone else